probes/toolSonarTypeLiftInstalled/impl.go

```diff
@@ -0,0 +1,41 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package toolSonarTypeLiftInstalled
```

Comment:
typo.

but same comment (for all of these probes), so they need to be a separate pacakage?

Reply:
see comment above.